SPECIFICATION.md

```diff
@@ -0,0 +1,293 @@
+# Playbook specification
+
+This document describes the exact format required by the Ansible playbook verifier.
+
+```yaml
+---
+- name: Example playbook
+  hosts: localhost
+  become: yes
+  vars:
+    insights_signature_exclude: /hosts,/vars/insights_signature
+    insights_signature: !!binary |
+      base64 binary blob containing the embedded GPG signature
+  tasks:
+    - name: Display debug message
+    - ansible.builtin.debug:
+        msg: The signed playbook says hello!
+```
+
+## Only one playbook may be present
+
+In case there are multiple playbooks present in the file, only the first one will be verified. Any other playbook is explicitly ignored.
+
+**Warning:** Even though only the first playbook is verified, all of them may be emitted to standard output after the verification.
```

Comment:
"all of them may", or "all of them will"? since this is a specification, it ought to be clear on the behaviour